integration/init_test.go

```diff
@@ -79,6 +89,15 @@ func TestIntegration(t *testing.T) {
 
 	SetDefaultEventuallyTimeout(5 * time.Second)
 
+	buf := bytes.NewBuffer(nil)
+	cmd := pexec.NewExecutable("pack")
+	Expect(cmd.Execute(pexec.Execution{
```

Comment:
I think occam has built-in support for inspecting the builder which simplifies this code:

https://github.com/paketo-buildpacks/occam/blob/25c0b7b6a49f7c9d114b4544b7cb2057932d90d6/pack.go#L234

Reply:
Oh excellent!